pkg/api/validation/validation.go

```diff
@@ -1538,6 +1548,15 @@ func ValidatePersistentVolumeClaimSpec(spec *api.PersistentVolumeClaimSpec, fldP
 			allErrs = append(allErrs, field.Invalid(fldPath.Child("storageClassName"), *spec.StorageClassName, msg))
 		}
 	}
+	if spec.VolumeMode != nil && *spec.VolumeMode == api.PersistentVolumeBlock && !utilfeature.DefaultFeatureGate.Enabled(features.BlockVolumeSupport) {
```

Comment:
When the feature is disabled then no value of this field is valid, even "Filesystem"